homeassistant/components/tuya/translations/select.en.json

```diff
@@ -48,6 +48,38 @@
             "on": "On",
             "power_off": "Off",
             "power_on": "On"
+        },
+        "tuya__vacuum_cistern": {
+            "low": "Low",
+            "middle": "Middle",
+            "high": "High",
+            "closed": "Closed"
+        },
+        "tuya__vacuum_collection": {
+            "small": "Small",
+            "middle": "Middle",
+            "large": "Large"
+        },
+        "tuya__vacuum_mode": {
+            "standby": "Standby",
+            "random": "Random",
+            "smart": "Smart",
+            "wall_follow": "Wall Follow",
+            "mop": "Mop",
+            "spiral": "Spiral",
+            "left_spiral": "Left Spiral",
+            "right_spiral": "Right Spiral",
+            "bow": "Bow",
+            "left_bow": "Lef Bow",
+            "right_bow": "Right Bow",
+            "partial_bow": "Partial Bow",
+            "chargego": "Charge Go",
```

Comment:
This is return to dock or something?

Reply:
Yes, it's the return to dock function, it includes why Tuya's documentation does (https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo).

In practice, it happens that some devices have a `switch_charge` switch and others use `mode chargego`.

Reply:
Right should be call is `Go Charge` in that case? Or `Charging`?

Reply:
> 

I changed it to Return to dock, ok?